hack/run-test.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+readonly CURRENT_SCRIPT=$(basename $0)
+
+HEADER_MESSAGE="Running Tests "
+
+usage() {
+    print "Usage:"
+    print "  ${CURRENT_SCRIPT} [-h] -s suites -p extraparams [-m header_message]"
+    print ""
+    print "Options:"
+    print "    -h                Help for ${CURRENT_SCRIPT}"
+    print "    -t                list of space separated paths to Testsuites to execute"
+    print "    -p                string with extra Params for ginkgo"
+    print "    -m                Message header to print before test execution. Default '${HEADER_MESSAGE}'"
+}
+
+exit_error() {
+    print ""
+    print "error: $*"
+    exit 1
+}
+
+print() {
+    echo "$*" >&2
+}
+
+HEADER_MESSAGE="Running Tests "
+
+main() {
+    while getopts ':ht:p:m:' OPT; do
+        case "$OPT" in
+        h)
+            usage
+            exit 0
+            ;;
+        t)
+            GINKGO_SUITS="${OPTARG}"
+            ;;
+        p)
+            EXTRA_PARAMS="${OPTARG}"
+            ;;
+        m)
+            HEADER_MESSAGE="${OPTARG}"
+            ;;
+        ?)
+            usage
+            exit_error "invalid argument: ${OPTARG}"
+            ;;
+        esac
+    done
+    shift $((OPTIND - 1))
+
+    if [ -z "$GINKGO_SUITS" ] || [ -z "$EXTRA_PARAMS" ]; then
+        usage
+        exit_error "Missing arguments -t or -p"
+    fi
+
+    LATENCY_TEST_RUN=${LATENCY_TEST_RUN:-"false"}
+
+    which ginkgo
+    if [ $? -ne 0 ]; then
+        echo "Downloading ginkgo tool"
+        go install -mod=mod github.com/onsi/ginkgo/v2/ginkgo@v2.6.1
+        ginkgo version
+    fi
+
+    NO_COLOR=""
+    if ! which tput &>/dev/null 2>&1 || [[ $(tput -T$TERM colors) -lt 8 ]]; then
+        echo "Terminal does not seem to support colored output, disabling it"
+        NO_COLOR="-noColor"
```

Comment:
just realized this is out of date with ginkgo v2, should be `--no-color` now IIRC

Reply:
Changed. 
Thanks for the catch